var/spack/repos/builtin/packages/hpx/package.py

```diff
@@ -83,16 +83,23 @@ class Hpx(CMakePackage, CudaPackage):
     # https://github.com/STEllAR-GROUP/hpx/issues/4728#issuecomment-640685308
     depends_on('boost@:1.72.0', when='@:1.4')
 
+    depends_on('boost@1.64.0:', when='@1.5')
+    depends_on('boost@1.66.0:', when='@1.6')
+
```

Comment:
These don't 100% belong here so I can move them to a separate PR if that's easier.

Reply:
Let's keep it here